docs/features.md

```diff
@@ -110,6 +110,17 @@ The document lists and describes all the feature flags that are available in Kym
   },
   ```
 
+- **DISABLED_NODES** - an array of IDs of navigation nodes that are hidden from navigation. Format: `<category>.<nodeName>` or `<namespace>.<category>.<nodeName>`).
```

Comment:
I know it's not your code, it's been here before, but I think the pattern here is not correct. It suggests that the `<namespace>` is a variable when it's not.

```suggestion
- **DISABLED_NODES** - an array of IDs of navigation nodes that are hidden from navigation. Format: `<category>.<nodeName>` or `namespace.<category>.<nodeName>`).
```